.release/linux/postrm

```diff
@@ -2,7 +2,9 @@
 
 if [ "$1" = "purge" ]
 then
-  userdel vault
+	if command -v userdel &> /dev/null; then
+		userdel vault
```

Comment:
Should we support deluser and adduser &co which are present on some other systems?

Reply:
Yeah that might be a good addition, not sure if I'm really happy with this fix overall though. 

It would probably be clearer overall to have dedicated vault-cli rpm/dep packages in the end. This feels like I'm hiding issues down the line for people that do want to run vault as a server in minimal containers.